src/main/io/msp_protocol.h

```diff
@@ -64,17 +64,16 @@
 #define API_VERSION_LENGTH                  2
 
 #define MULTIWII_IDENTIFIER "MWII";
-#define CLEANFLIGHT_IDENTIFIER "CLFL"
-#define BETAFLIGHT_IDENTIFIER "BTFL"
 #define BASEFLIGHT_IDENTIFIER "BAFL";
+#define BETAFLIGHT_IDENTIFIER "BTFL"
+#define CLEANFLIGHT_IDENTIFIER "CLFL"
+#define INAV_IDENTIFIER "INAV"
+#define RACEFLIGHT_IDENTIFIER "RCFL"
```

Comment:
Isn't that just transfering the problem with different files to some other file? Can't this #define be set outside this file instead then? 
Maybe you already thought through every alternatives, I just thought it looked strange to define this non-betaflight identifiers here since they will not be used in the code (betaflight code that is).